app/src/main/java/com/pvp/app/Application.kt

```diff
@@ -117,6 +120,22 @@ class Application : Application(), Configuration.Provider {
         }
     }
 
+    private fun createTaskAutocompleteWorker(){
+        Log.e("AUTOCOMPLETE", "CREATING")
```

Comment:
Aren't these for debugging while developing? Change to `Log.d` for debug type instead of error type or remove.